lib/services/livesync/livesync-service.ts

```diff
@@ -3,11 +3,9 @@ import * as helpers from "../../common/helpers";
 import * as path from "path";
 import * as semver from "semver";
 import * as fiberBootstrap from "../../common/fiber-bootstrap";
-
-let gaze = require("gaze");
+let choki = require("chokidar");
```

Comment:
Yes, there is a reason. I observed that gaze is quite slow and unstable when watching for a large number of files, while chokidar handles this scenario fast and reliable. Chokidar is the preferred watching tool used in webpack.

I added the corresponding entry in package.json